contracts/assemblyscript/nep21-basic/main.ts

```diff
@@ -3,7 +3,7 @@ import { AccountId, Amount } from './types'
 import {
   allowanceRegistry,
   balanceRegistry,
-  getAllowanceKey as keyFrom,
+  getAllowanceKey as keyFrom, // why not keep the name the same?
```

Comment:
because it reads oh so nicely inline 🥰

Reply:
fair enough – at first I was thinking the exported function could just be called `keyFrom`, but on second look I like the more explicit name better